### PR TITLE
Run on first page load and ensure conversion review has proper behavior

### PR DESCRIPTION
**Pull request by garrettmflynn**
When going through a straight run of the SpikeGLX-Phy test pipeline, it was noticed that full and preview conversions were not being triggered. 

This PR fixes this behavior to ensure that a preview conversion is triggered on the first run _even if no manual changes have been made by the user_. The behavior of the Conversion Review page has also been updated to trigger a conversion before reaching DANDI Upload.

### CODE REVIEW

**Comment by CodyCBakerPhD** [comment]
Any way to add tests? (also some failing atm)

**Comment by garrettmflynn** [comment]
Gotcha thanks for flagging! Added a check to the tests so that `main` would fail and fixed how tests are handled (specifically, so that transitions are awaited properly before moving forward with the next test) so that these succeed

**Review by CodyCBakerPhD — APPROVED** [review]

**Comment by CodyCBakerPhD** [comment]
@garrettmflynn 2 pages in the storybook now fail to fetch - is this intended or do you know what's happening?

**Comment by garrettmflynn** [comment]
Found the issue. Fixed

**Review by CodyCBakerPhD — APPROVED** [review]

**Comment by CodyCBakerPhD** [comment]
@garrettmflynn Main tests failing again

File: src/renderer/src/stories/pages/Page.js [hunk 1]
```diff
@@ -254,9 +254,11 @@ export class Page extends LitElement {
 
     checkSyncState = async (info = this.info, sync = info.sync) => {
         if (!sync) return;
+        if (isStorybook) return;
 
         const { desyncedData } = info.globalState;
 
+        f
```
Comment (garrettmflynn) [inline src/renderer/src/stories/pages/Page.js]: Here's the culprit. Whoops!

**Comment by garrettmflynn** [comment]
Looks like one of my wonderful cats stepped on my keyboard and inserted a stray `f` in the script. 

My bad! Works locally now

**Comment by CodyCBakerPhD** [comment]
> Looks like one of my wonderful cats stepped on the keyboard and inserted a stray f in the script.

😂

**Pull request merged**